CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
I have updated the VERSION in the repository because I have updated to a new meta-resin release. But the purpose of this PR is not to do a Intel NUC release.
Yes it will trigger a new deployment to staging. But so would any other PR that gets merged